ldms/src/core/ldms.c

```diff
@@ -1627,18 +1627,24 @@ static size_t compute_set_sizes(const char *instance_name, ldms_schema_t schema,
 	return *meta_sz + *array_data_sz;
 }
 
-void __make_mdesc(ldms_mdesc_t vd, ldms_mdef_t md, off_t *value_off)
+void __make_mdesc(ldms_mdesc_t vd, ldms_mdef_t md, off_t *value_off, size_t remaining)
```

Comment:
Right before this function add:

#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wstringop-overflow"